catroid/src/main/java/org/catrobat/catroid/content/bricks/UserBrick.java

```diff
@@ -297,6 +297,17 @@ public void onLayoutChanged(View currentView) {
 		}
 	}
 
+	public void deleteUserVariables(String userVariableName) {
+		List<UserScriptDefinitionBrickElement> currentElements = getUserScriptDefinitionBrickElements();
+		for (int id = 0; id < currentElements.size(); id++) {
+			if (currentElements.get(id).getText().equals(userVariableName) && currentElements.get(id).isVariable()) {
+				Context brickContext = getDefinitionBrick().view.getContext();
+				getDefinitionBrick().removeVariablesInFormulas(currentElements.get(id).getText(), brickContext);
+				currentElements.remove(id);
```

Comment:
Well, I've deleted the method now anyway. It was already unused before - would possibly the best to refactor that method in the userbricks-branch in the future